{{cookiecutter.project_name}}/README.md

```diff
@@ -1,30 +1,41 @@
+The sample utilizes Github Actions, SuperLinter and Bicep.
+
+## CD Quickstart
+
+To enable the CD pipeline that deploys resources to Azure, you will need to create a Service Principal in Azure that will be used by the agent to authenticate and create resources. To do so follow these steps: [Use the Azure login action with a service principal secret](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows#use-the-azure-login-action-with-a-service-principal-secret)
 
 ## Structure example description
+
     .
-    ├── .github               # Github Action folders that enables for CI and CD for your repo
-    │   ├── actions           # Add custom actions
-    │   ├── scripts           # Add custom scripts
-    │   ├── workflows         # Build your Github Actions by adding a workflow file
-    ├── documents             # Documentation about the project
-    │   ├── design            # Add your designs and architectures here
-    │   ├── spikes            # Holds code samples for investigation
-    │   ├── adr               # ADR (Architecture Decision Record)
-    ├── assets                # Holds content like 'jpg', 'img', 'css', 'fonts's
-    ├── infrastructure        # Holds your infrastructure as code, in our case Terraform
-    │   ├── terraform         # Holds all terraform files
-    │   ├── variables         # Holds all terraform variables
-    ├── lib                   # Standard folder for third party solutions and libraries
-    ├── src                   # Source code folder that holds all the solutions
-    │   ├── python-solution
+    ├── .github                     # Github Actions folders for CI/CD
+    │   ├── actions                 # Custom actions
+    │   ├── linters                 # Linter exception files
+    │   ├── scripts                 # Custom shell scripts
+    │   ├── workflows               # Github Actions workflow files
+    ├── documents                   # Documentation about the project
+    │   ├── design                  # Designs and architectures
+    │   │   ├── assets              # Binary content like 'jpg', 'png', etc.
+    │   ├── spikes                  # Code and assets created during 'spikes' (tech investigation) 
+    │   ├── adr                     # ADRs (Architecture Decision Records)
+    │   │   ├── assets              # Binary content like 'jpg', 'png', etc.
+    ├── assets                      # Binary content like 'jpg', 'png', etc.
+    ├── infrastructure              # Infrastructure as Code (IaC) templates
+    │   ├── bicep                   # IaC Bicep files
+    │   │   ├── modules             # Bicep main modules (e.g. infrastructure)
+    │   │   ├── resource-templates  # Azure resource templates (e.g. storage)
+    │   │   ├── variables           # Bicep deployment common and environment parameters 
+    ├── lib                         # Standard folder for third party solutions and libraries
```

Comment:
i think lib folder missing from folder structure

Reply:
Good catch! The lib was part of .gitignore so it wasn't syncing. Fixed now :)